src/azul/service/async_manifest_service.py

```diff
@@ -25,6 +28,20 @@
 logger = logging.getLogger(__name__)
 
 
+class Status(Enum):
+    running = ()
+    succeeded = ()
+    failed = ()
+    timed_out = ()
+    aborted = ()
+
+    def __init__(self):
+        super().__init__()
+        name = self._name_.upper()
+        self._value_ = name
+        self._member_map_[name] = self
```

Comment:
What's this for?

Reply:
This allows for the getting at the Status enumeration item like a key on a dict. For instance https://github.com/DataBiosphere/azul/pull/3337/commits/010ebd60324293a0719d79f4d8cd603b1b4c80fd#diff-898948d91fb9583e94bba2ddbe7a303c7b6f2e466d97947f2efddd60ecc4f78aR104.